### PR TITLE
Editionalising onward journeys on amp

### PR DESCRIPTION
**Pull request by NataliaLKB**
This will make sure that the edition is no longer being hardcoded, and is actually coming from the user's preferences/location.

cc @stephanfowler

### CODE REVIEW

File: facia/app/controllers/FaciaController.scala [hunk 1]
```diff
@@ -93,10 +93,12 @@ trait FaciaController extends Controller with Logging with ExecutionContexts wit
     }
   }
 
-  def renderSomeFrontContainersMf2(rawNum: String, rawOffset: String, path: String) = MemcachedAction { implicit request =>
-    def getEditionFromString(edition: String) = Edition.all.find(_.id.toLowerCase() == "int").getOrElse(Edition.all.head)
+  def renderSomeFrontContainersMf2(rawNum: String, rawOffset: String, section: String) = MemcachedAction { implicit request =>
+    val edition = Edition(request)
+    val sectionsToLoad = List("commentisfree", "sport", "football", "fashion", "lifeandstyle", "education", "culture", "business", "technology", "politics", "environment", "travel", "film", "media", "money", "society", "science", "music", "books", "stage", "cities", "tv-and-radio", "artanddesign", "global-development")
```
Comment (jamesgorrie) [inline facia/app/controllers/FaciaController.scala]: Perhaps a `TODO` to standardise this list?

**Comment by jamesgorrie** [comment]
:+1:

File: facia/app/controllers/FaciaController.scala (hunk 1 above)
Comment (NataliaLKB) [inline facia/app/controllers/FaciaController.scala]: Yes you are completely correct!! Will add that now. :sparkles:

**Comment by NataliaLKB** [comment]
reverting to deploy on Monday (instead of a friday night!)

**Pull request merged**